pkg/asset/machines/master.go

```diff
@@ -177,6 +178,16 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 			return errors.Wrap(err, "failed to create master machine objects")
 		}
 		azure.ConfigMasters(machines, clusterID.InfraID)
+	case baremetaltypes.Name:
+		// FIXME: baremetal
```

Comment:
can we track the fixme in JIRA or github issues rather than code.

Reply:
I think this is a rebase mistake and can be removed.

See https://github.com/openshift-metal3/kni-installer/commit/1128a2ab244ff52d84fa81c89633bf5e0c3c5af2

And it was removed in https://github.com/openshift-metal3/kni-installer/commit/5d1f9a8e364e6d2bd0a9d17b90bc86216d1a36ed#diff-66b4febb36af0238bb6c09e72c1c44ec

Reply:
Sure, I can clean these up.

There's a few like this one https://github.com/openshift/installer/pull/1873/files#diff-b69d29c6306c28bb385391cd0af1b7fcR81 that have descriptive information and a link to a related issue, can we leave those in code? I feel like it's harder to lose track of it when it's inline.

But happy to remove them all and create issues for all of them if you prefer.